From 94261b84415ba0de27c4d3a8686e2edf74a01ce9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Hundeb=C3=B8ll?= <mnhu@prevas.dk>
Date: Mon, 31 Aug 2015 13:32:16 +0200
Subject: [PATCH] avoid thin static archives

GYP builds thin archives by default on platforms that support it (the
GNU toolchain on Linux does, don't think OS X does.)  disable
it by setting standalone_static_library: 1 in the v8.gyp file
---
 tools/gyp/v8.gyp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/gyp/v8.gyp b/tools/gyp/v8.gyp
index 242ddf0..b43486f 100644
--- a/tools/gyp/v8.gyp
+++ b/tools/gyp/v8.gyp
@@ -107,6 +107,7 @@
     {
       'target_name': 'v8_snapshot',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'conditions': [
         ['want_separate_host_toolset==1', {
           'toolsets': ['host', 'target'],
@@ -179,6 +180,7 @@
     {
       'target_name': 'v8_nosnapshot.<(v8_target_arch)',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'dependencies': [
         'v8_base.<(v8_target_arch)',
       ],
@@ -236,6 +238,7 @@
     {
       'target_name': 'v8_base.<(v8_target_arch)',
       'type': 'static_library',
+      'standalone_static_library': 1,
       'variables': {
         'optimize': 'max',
       },
-- 
2.5.1

